thin-replica-server/proto/thin_replica.proto

```diff
@@ -129,3 +129,10 @@ message EventGroup {
   google.protobuf.Timestamp record_time = 3;
```

Comment:
This field is supposed to be set by the TRS and hence the TRC shouldn't need to try finding the time from the list of events.